gen/azure/cloud-config.yaml

```diff
@@ -74,7 +76,7 @@ bootcmd:
 disk_setup:
   ephemeral0:
     table_type: gpt
-    layout: [ 50, 50 ]
+    layout: [ 25, 50, 25 ]
```

Comment:
I'd like to keep the allocation between docker and mesos partitions to a 50/50 split... so can we change this to: `layout: [ 48, 48, 4 ]`? With the default ephemeral disk size of 50GB, this will give users 2GB of tmp space, which should be plenty of room for the download and extract of DC/OS and the deb packages (~700MB required).

Also, @JackQuincy do you know if it is possible for users to encounter ephemeral disk sizes smaller than 50GB? If so, we may need to increase the percentage for /var/tmp.